README.md

```diff
@@ -43,6 +43,36 @@ yarn dev
 rails s
 ```
 
+## Postgres setup
+
+```bash
+brew services start postgresql
```

Comment:
First we need to install it. Most cases one will have it installed from other projects, but still.
`brew install postgresql`